.github/workflows/test_and_deploy.yml

```diff
@@ -0,0 +1,126 @@
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup conda env
+        run: |
+          source "$CONDA/etc/profile.d/conda.sh"
+          conda config --set always_yes yes --set changeps1 no
+          conda update -q conda
+          conda info -a
+          conda install -q conda-build
+          conda create -n numba-scipy -c numba python=${{ env.MAIN_PYVER }} numba>=0.45 'scipy>=0.16,<1.4' flake8 pytest
+          conda activate numba-scipy
+          pip install -e .
+      - name: Lint with flak8e
+        run: |
+          source "$CONDA/etc/profile.d/conda.sh"
+          conda activate numba-scipy
+          flake8 numba_scipy
+      - name: Pytest
+        run: |
+          source "$CONDA/etc/profile.d/conda.sh"
+          conda activate numba-scipy
+          bash ./buildscripts/incremental/test.sh
+      ##- name: Coveralls
+      ##  env:
+      ##    GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      ##  run: |
+      ##    source "$CONDA/etc/profile.d/conda.sh"
+      ##    conda activate mg
+      ##    conda install -c conda-forge coveralls
+      ##    coveralls --service=github
+      - name: Conda Build
+        run: |
+          source "$CONDA/etc/profile.d/conda.sh"
+          conda build -c defaults -c numba --python ${{ env.MAIN_PYVER }} buildscripts/conda_recipes/numba-scipy/
+          # This doesn't rebuild, but simply computes the name of the file that was previously built
+          OUTPUT=$(conda build --output -c defaults -c numba --python ${{ env.MAIN_PYVER }} buildscripts/conda_recipes/numba-scipy/)
+          echo "Path to built package:"
+          echo $OUTPUT
+          echo "CONDA_BUILD_OUTPUT=$OUTPUT" >> $GITHUB_ENV
+      - name: Upload build artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: built_package
+          path: ${{ env.CONDA_BUILD_OUTPUT }}
+          retention-days: 7
+
+  test_pyver:
+    runs-on: ubuntu-latest
+    needs: build
+    strategy:
+      matrix:
+        pyver:
+          - 3.6
+          - 3.7
+          - 3.8
+    steps:
+      - name: Download build artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: built_package
+          path: ./artifact_storage
+      - name: Setup conda env
+        run: |
+          source "$CONDA/etc/profile.d/conda.sh"
+          conda config --set always_yes yes --set changeps1 no
+          conda update -q conda
+          conda create -n numba-scipy -c numba python=${{ matrix.pyver }} numba>=0.45 'scipy>=0.16,<1.4' flake8 pytest
+          conda activate numba-scipy
+          # Install built_package
+          BUILT_PKG=$(ls ./artifact_storage | head -1)
+          conda install ./artifact_storage/$BUILT_PKG
+          # Install dependencies listed in built package
+          # (This is a workaround because conda does not install dependencies for local file installs)
+          conda update -c defaults -c conda-forge numba-scipy
```

Comment:
Is this going to end up with confusing state(s) given the `create` command a few lines up installs _some_ dependencies too?

Reply:
I don't know, I copied this from Metagraph.

Reply:
but probably should not use `conda-forge` --- that is just asking for trouble in this case.

Reply:
Here is the log:

```
## Package Plan ##

  environment location: /usr/share/miniconda/envs/numba-scipy

  added / updated specs:
    - numba-scipy


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    numba-scipy-0.2.0          |             py_0           9 KB  numba
    ------------------------------------------------------------
                                           Total:           9 KB

The following packages will be UPDATED:

  numba-scipy        <unknown>::numba-scipy-0.1.0dev0-py_66 --> numba::numba-scipy-0.2.0-py_0



Downloading and Extracting Packages

numba-scipy-0.2.0    | 9 KB      |            |   0% 
numba-scipy-0.2.0    | 9 KB      | ########## | 100% 
numba-scipy-0.2.0    | 9 KB      | ########## | 100% 
Preparing transaction: ...working... done
Verifying transaction: ...working... done
Executing transaction: ...working... done
```

Reply:
I think it will be fine for the time being.

Reply:
ok, it turns out, that this is actually not OK. The update actually overwrites the installed blob from the build stage with a 0.2.0 package from anaconda.org. This is because the 0.2.0 tag was on the release branch and 0.1.0 on `master`. So even though the version being built is technically beyond 0.2.0, it has the wrong version. Removing the update step means installing the correct package, however the incorrect versioning is something to address in a followup issue.